primitives/src/lib.rs

```diff
@@ -89,7 +89,7 @@ pub type BridgeId = (ChainNonce, ChainNonce);
 
 pub const SECONDS_PER_YEAR: Timestamp = 365 * 24 * 60 * 60;
 
-pub type LeasePeriod = BlockNumber;
+pub type LeasePeriod = u32;
```

Comment:
on relaychain `LeasePeriod` is BlockNumber, and we use the same BlockNumber like in relaychain (u32), so it should be fine maybe?